### PR TITLE
Header fixes and some tidying

### PR DESCRIPTION
**Pull request by AlecRust**
Fixes top bar now showing menu items at all screen widths, on "web log" and "list server" pages.

image

image

Also:
- Fix Stylelint pre-commit hook preventing commit of minified CSS
- Fix Stylelint not linting all CSS source files
- Fix JS errors on "web log" and "list server" pages
- Specify Yarn v3 package manager in package.json 

cc @jakobbouchard

### CODE REVIEW

**Comment by imjuniper** [comment]
I added a `.stylelintignore` file in my docs PR, I forgot it in the original PR I made haha. I'll merge both of our files once our PRs are merged.

**Review by jaapmarcus — APPROVED** [review]

**Comment by jaapmarcus** [comment]
@AlecRust  please fix the merge conflicts if have time.

**Pull request merged**
